ci/submodule-sync.sh

```diff
@@ -70,24 +70,18 @@ echo "Test against ${cudf_sha}..."
 
 MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -B"
 set +e
-${MVN} verify ${MVN_MIRROR} \
+# Don't do a full build. Just try to update/build CUDF with no patches on top of it.
+${MVN} validate ${MVN_MIRROR} \
```

Comment:
I think we care only about one execution in the `validate` phase. So we could

```suggestion
${MVN} antrun:run@build-libcudf ${MVN_MIRROR} \
```